### PR TITLE
Feature overhaul jitsi meet

### PR DESCRIPTION
**Pull request by BacLuc**
Issue: https://github.com/sopra-fs22-group-36/screw-your-neighbor-server/issues/12

### CODE REVIEW

**Review by MorisCamporesi — APPROVED** [review]
looks good to me, had no issues during testing

File: src/components/views/game/game/videochat/VideoChat.tsx [hunk 1]
```diff
@@ -69,6 +69,17 @@ export const VideoChat = observer(() => {
                 iframe.style.width = "100%"
               }}
             />
+
+            <div> </div>
```
Comment (BacLuc) [inline src/components/views/game/game/videochat/VideoChat.tsx:73]: These div should be unnecessary.
Spaces between components should be done with padding and margin.

File: src/components/views/game/game/videochat/VideoChat.tsx [hunk 2]
```diff
@@ -69,6 +69,17 @@ export const VideoChat = observer(() => {
...
+            <Button
```
Comment (BacLuc) [inline src/components/views/game/game/videochat/VideoChat.tsx:74]: very simple, funcionality is here

**Review by BacLuc — COMMENTED** [review]
Next time approve and merge my PR and make a new one

**Comment by MorisCamporesi** [comment]
> Next time approve and merge my PR and make a new one

Yes absolutely, I checked out the wrong branch and then realized after push that it would fall into the same PR.

File: src/components/views/game/game/videochat/VideoChat.tsx (hunk 1 above)
Comment (MorisCamporesi) [reply src/components/views/game/game/videochat/VideoChat.tsx:73]: 🙃 absolutely, I _would_ actually know that

**Pull request merged**